Gemfile

```diff
@@ -3,6 +3,6 @@ source 'https://rubygems.org'
 gemspec
 
 group :test, :development do
-  gem 'rake', '~> 10.4.2'
   gem 'minitest', '~> 5.8.2'
+  gem 'rake', '~> 10.4.2'
```

Comment:
Note: this should be `'~> 10,4', '>= 10.4.2'` so you could get API-compatible versions of 10.x